taui/src/components/neighborhood-details.js

```diff
@@ -99,13 +99,12 @@ class NeighborhoodDetails extends PureComponent<Props> {
     const maxSubsidy = neighborhood.properties['max_rent_' + rooms + 'br'] || '–––'
 
     return (
-      <div className='neighborhood-details__stats'>
-        <div className='neighborhood-details__rent'>
+      <div className='neighborhood-details__rent'>
+        <div>
           <div className='neighborhood-details__rent-label'>{t('NeighborhoodDetails.MaxRent')}</div>
-          <div className='neighborhood-details__rent-value'>${maxSubsidy}</div>
           <div className='neighborhood-details__rent-rooms'>{rooms}{t('NeighborhoodDetails.BedroomAbbr')}</div>
         </div>
-        <NeighborhoodListInfo neighborhood={neighborhood} />
+        <div className='neighborhood-details__rent-value'>${maxSubsidy.toLocaleString("en-US")}</div>
```

Comment:
This and the instance below should adjust depending on the user's chosen locale because different locales use different number formatting.

I think it should be sufficient to add `i18n` to line 97: `const {t, i18n} = useTranslation()`

and then to adjust this to be: `maxSubsidy.toLocaleString(i18n.language)`

If it turns out to be substantially more complicated than that then I'd ignore it because anyone using the app is most likely familiar with US number formatting conventions (because they live here, and number formatting is not hard to adjust to, in my experience).